pipeline/static/add_product/add_product.js

```diff
@@ -0,0 +1,56 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Great job splitting this out into a separate file, makes it more decluttered and more readable